x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/QueryRulesIndexService.java

```diff
@@ -256,8 +257,7 @@ public void listQueryRulesets(int from, int size, ActionListener<QueryRulesetRes
             final SearchSourceBuilder source = new SearchSourceBuilder().from(from)
                 .size(size)
                 .query(new MatchAllQueryBuilder())
-                .docValueField(QueryRuleset.ID_FIELD.getPreferredName())
-                .storedFields(Collections.singletonList("_none_"))
+                .fetchSource(new String[] { QueryRuleset.ID_FIELD.getPreferredName(), QueryRuleset.RULES_FIELD.getPreferredName() }, null)
```

Comment:
Note: I deviated from the `docValueField` that we used in previous CRUD APIs to use the `source` because `rules` is an object field. I'm open to alternate solutions if this isn't desired, but it felt OK.